openfisca_us/variables/hhs/tanf.py

```diff
@@ -121,6 +142,33 @@ class tanf_total_gross_income(Variable):
     unit = USD
     reference = "https://www.dhs.state.il.us/page.aspx?item=15814"
 
+    def formula(spm_unit, period, parameters):
+        return add(spm_unit, period, "tanf_gross_earned_income", "tanf_gross_unearned_income")
+
+class tanf_gross_earned_income(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    label = "TANF gross earned income"
+    documentation = "Gross earned income for calculating Temporary Assistance for Needy Families benefit."
+    unit = USD
+    reference = "https://www.dhs.state.il.us/page.aspx?item=15814"
+
+    def formula(spm_unit, period, parameters):
+        return spm_unit.sum(spm_unit.members("market_income", period))
+
+
+class tanf_gross_unearned_income(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    label = "TANF gross unearned income"
+    documentation = "Gross unearned income for calculating Temporary Assistance for Needy Families benefit."
+    unit = USD
+    reference = "https://www.dhs.state.il.us/page.aspx?item=15814"
+
+    def formula(spm_unit, period, parameters):
+        return add(spm_unit, period, "ssi")
```

Comment:
This could be (no functional difference, just style): 
```suggestion
        return spm_unit("ssi", period)
```
Or, we could make a policy parameter for the list of income sources for unearned TANF gross income, but that seems unnecessary.